src/app/register/register.component.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../styles.scss';
+
+.custom-group{
```

Comment:
Shouldn't we re-use this style from a common place in both `login` and the `register` component?

Reply:
done